deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,124 @@ spec:
+    ```
+    $ kubectl get po -n <namespace>
+      NAME                               READY   STATUS    RESTARTS   AGE
+      kubefed-operator-db58d9b89-8694x   1/1     Running   0          15s
+
+    ```
+    * Create a `KubeFedWebHook` resource to instatiate an admission webhook controller for kubefed. It's recommended to install this CR with `scope:Cluster`
+    to allow webhook controller to serve API requests coming from all `kubefed` control planes within the same cluster.
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFedWebHook
+    metadata:
+      name: kubefed-webhook-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    * Create a `KubeFed` resource to drive the installation of `KubeFed` setting the scope to be either `Namespaced` or `Cluster`. 
+    If you're planning to federate a cluster-scoped resource type like `StorageClass` then set the scope field to `Cluster`
+    
+    ```
+    $ cat <<-EOF | kubectl apply -n <namespace> -f -
+    ---
+    apiVersion: operator.kubefed.io/v1alpha1
+    kind: KubeFed
+    metadata:
+      name: kubefed-resource
+    spec: 
+      scope: Cluster 
+    ---
+    ```
+    ```
+    $ kubectl get po -n <namespace> 
+    NAME                                          READY   STATUS    RESTARTS   AGE
+    kubefed-admission-webhook-77f7bd89dd-92kzj    1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-5t2fk   1/1     Running   0          25s
+    kubefed-controller-manager-67d5d5cc99-b865z   1/1     Running   0          25s
+    kubefed-operator-db58d9b89-8694x              1/1     Running   0          61s
+    
+    ```
+    **Important Note**: 
+    * It's recommended to deploy kubefed in either a namespace-scoped or cluster-scoped mode but not both ways as deploying a cluster-scoped and 
+    namespace-scoped kubefed control plane to the same cluster will result in both control planes try to manage resources in the namespace of the namespace-scoped 
+    control plane and you will not see the desired outcome. For more information, use this [link](https://github.com/openshift/federation-dev/blob/master/docs/kubefed-scope.md) 
 
-    **Important Note**: The instructions in this guide refer to the namespace
-    where federation is installed as `<namespace>` in command examples. The
-    upstream user guide uses `kube-federation-system` as the name of this namespace.
+    * To add federation capability to your namespace, you are only required to install kubefed in the hosting cluster; 
+    there is no need to install kubefed in the target clusters (and no effect gained from doing so).
 
-    ### Get the kubefedctl CLI tool
+    * The instructions in this guide refer to the namespace where `KubeFed` is installed as `<namespace>` in command examples. 
+    The upstream user guide uses `kube-federation-system` as the name of this namespace.
```

Comment:
Does this differ downstream?

Reply:
Need to confirm this info before I say anything here.